etc/generate-envoy-config/envoy.libsonnet

```diff
@@ -0,0 +1,201 @@
+{
+  bootstrap(listeners, clusters): {
+    admin: {
+      access_log: [
+        {
+          name: 'envoy.access_loggers.stderr',
+          typed_config: {
+            '@type': 'type.googleapis.com/envoy.extensions.access_loggers.stream.v3.StderrAccessLog',
+          },
+        },
+      ],
+      address: {
+        socket_address: {
+          address: '0.0.0.0',
+          port_value: 9901,
+        },
+      },
+    },
+    static_resources: {
+      clusters: clusters,
+      listeners: listeners,
+    },
+    overload_manager: {
+      actions: [
+        {
+          name: 'envoy.overload_actions.shrink_heap',
+          triggers: [
+            {
+              name: 'envoy.resource_monitors.fixed_heap',
+              threshold: {
+                value: 0.95,
+              },
+            },
+          ],
+        },
+        {
+          name: 'envoy.overload_actions.stop_accepting_requests',
+          triggers: [
+            {
+              name: 'envoy.resource_monitors.fixed_heap',
+              threshold: {
+                value: 0.98,
+              },
+            },
+          ],
+        },
+      ],
+      refresh_interval: '0.25s',
+      resource_monitors: [
+        {
+          name: 'envoy.resource_monitors.fixed_heap',
+          typed_config: {
+            '@type': 'type.googleapis.com/envoy.extensions.resource_monitors.fixed_heap.v3.FixedHeapConfig',
+            max_heap_size_bytes: 500000000,
+          },
+        },
+      ],
+    },
+    layered_runtime: {
```

Comment:
what does this do?

Reply:
Basically you get a warning in the logs if you don't have this.  "Runtime" is their buzzword for "configuration".  The runtime has layers that can be read from this file, other files on disk (like a configmap, and it will update without having to restart envoy and drain traffic), or a service discovery server.  We are just using it to set a limit on connections to something less than infinity.